src/opm/material/fluidmatrixinteractions/EclMaterialLawManagerInitParams.cpp

```diff
@@ -52,11 +52,12 @@ InitParams(EclMaterialLawManager<Traits>& parent, const EclipseState& eclState,
 template <class Traits>
 void
 EclMaterialLawManager<Traits>::InitParams::
-run() {
+run(const std::function<std::vector<int>(const FieldPropsManager&, const std::string&, const unsigned int, bool)>&
+    getFieldPropIntOnLeafAssigner) {
```

Comment:
Should we strip the get here and use e.g. fieldPropIntOnLeafAssigner? Same elsewhere.

Reply:
This is the same name (except for the _) of the new member functions in EclProblem. I can get rid of the get, here, and create that inconsistency. Or, I can get rid of the get here and also in the new member functions in EclProblem.

Reply:
Done! (no get left)